stripe/functions/index.js

```diff
@@ -63,6 +63,22 @@ exports.createStripeCustomer = functions.auth.user().onCreate(event => {
   });
 });
 
+// Add a payment source (card) for a user by writing a stripe payment source token to Realtime database
+exports.addPaymentSource = functions.database.ref('/stripe_customers/{userId}/cards/{pushId}/token').onWrite(event => {
```

Comment:
At [this line](https://github.com/rldaulton/functions-samples/blob/916071aa75bf1a9d75c0c6c2e1a0e7956e9b881f/stripe/functions/index.js#L35), correct?